EventFilter/RPCRawToDigi/plugins/RPCTwinMuxDigiToRaw.cc

```diff
@@ -95,7 +95,7 @@ void RPCTwinMuxDigiToRaw::produce(edm::Event& event, edm::EventSetup const& setu
 
   std::map<int, FEDRawData> fed_data;
   // Loop over the FEDs
-  for (std::pair<int, std::vector<RPCAMCLink> > const& fed_amcs : fed_amcs_) {
+  for (std::pair<int, std::vector<RPCAMCLink> > const fed_amcs : fed_amcs_) {
```

Comment:
this is strange.
why is a simple const OK here?  Isn't this going to trigger a copy of a vector?

Reply:
Most of this fixes are suggested by the compiler. It says you need copy here or your need a reference here, in this case it said remove the reference
https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc7_amd64_gcc820/CMSSW_11_1_CLANG_X_2020-04-27-2300/EventFilter/RPCRawToDigi

Reply:
@mrodozov , the correct fux is to use `std::pair<const int`

Reply:
@mrodozov , please remove this fix. It is already part of https://github.com/cms-sw/cmssw/pull/29572/files#diff-d409478036411a5bded20820a924b87a